### ISSUE TITLE
Unable to build the project after installing react-native-document-picker

### ISSUE THREAD

**Issue by nguyenares** [microsoft/react-native-windows#8322]
Hello, recently I'm using react-native-windows to create a desktop application. My application need to allow users to select csv or excel file from their computers. Today I ran into a problem after installing [react-native-document-picker](https://github.com/rnmods/react-native-document-picker). Before installing RNDP, my project built successfully and I was able to run it. After install RNDP, my project was unable to build. So I make a test project to test it out, after those commands below, the build errors occur. How can I solve it? is there any other library to handle selecting file? Thank you.

**Commands to reproduce the issue:**
npx react-native init TestProject --template react-native@^0.64.0
cd TestProject
npx react-native-windows-init --overwrite
npm i --save react-native-document-picker
npx react-native run-windows --logging

**Build errors I get:**
```
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11
01: Unable to find package Humanizer. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\MyLan\Domino-Label-Printing\Test\TestProject\wind
ows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11 
01: Unable to find package Microsoft.CodeAnalysis.CSharp. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\MyLan\Domino-Label-Printing\T 
est\TestProject\windows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11 
01: Unable to find package Microsoft.CodeAnalysis.CSharp.Workspaces. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\MyLan\Domino-Label 
-Printing\Test\TestProject\windows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11 
01: Unable to find package RuntimeContracts. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\MyLan\Domino-Label-Printing\Test\TestProje 
ct\windows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed\Microsoft.ReactNative.Managed.csproj : warning NU1603: Microsoft. 
ReactNative.Managed depends on Microsoft.NETCore.UniversalWindowsPlatform (>= 6.2.9) but Microsoft.NETCore.UniversalWindowsPlatform 6.2.9 was not found. An approximate best matc 
h of Microsoft.NETCore.UniversalWindowsPlatform 6.2.12 was resolved. [D:\MyLan\Domino-Label-Printing\Test\TestProject\windows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11
02: Unable to find package System.Private.Uri with version (>= 4.3.2) [D:\MyLan\Domino-Label-Printing\Test\TestProject\windows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11 
02:   - Found 2 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.1 ] [D:\MyLan\Domino-Label-Printing\Test\TestProject\windows\testproject.sln]       
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11 
01: Unable to find package Microsoft.NETCore.App.Runtime.win-x64. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\MyLan\Domino-Label-Pr 
inting\Test\TestProject\windows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11 
01: Unable to find package Microsoft.NETCore.App.Runtime.win-x86. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\MyLan\Domino-Label-Pr 
inting\Test\TestProject\windows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11
01: Unable to find package Microsoft.WindowsDesktop.App.Runtime.win-x64. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\MyLan\Domino-L 
abel-Printing\Test\TestProject\windows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11 
01: Unable to find package Microsoft.WindowsDesktop.App.Runtime.win-x86. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\MyLan\Domino-L 
abel-Printing\Test\TestProject\windows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11 
01: Unable to find package Microsoft.AspNetCore.App.Runtime.win-x64. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\MyLan\Domino-Label 
-Printing\Test\TestProject\windows\testproject.sln]
D:\MyLan\Domino-Label-Printing\Test\TestProject\node_modules\react-native-windows\Microsoft.ReactNative.Managed.CodeGen\Microsoft.ReactNative.Managed.CodeGen.csproj : error NU11 
01: Unable to find package Microsoft.AspNetCore.App.Runtime.win-x86. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\MyLan\Domino-Label 
-Printing\Test\TestProject\windows\testproject.sln]
```

**My package.json**
```
{
  "name": "testproject",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "android": "react-native run-android",
    "ios": "react-native run-ios",
    "start": "react-native start",
    "test": "jest",
    "lint": "eslint .",
    "windows": "react-native run-windows"
  },
  "dependencies": {
    "react": "17.0.1",
    "react-native": "0.64.0",
    "react-native-document-picker": "^5.2.0",
    "react-native-windows": "^0.64.0-0"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9",
    "@babel/runtime": "^7.12.5",
    "@react-native-community/eslint-config": "^2.0.0",
    "babel-jest": "^26.6.3",
    "eslint": "7.14.0",
    "jest": "^26.6.3",
    "metro-react-native-babel-preset": "^0.64.0",
    "react-test-renderer": "17.0.1"
  },
  "jest": {
    "preset": "react-native"
  }
}
```

**Comment by asklar** [comment]
@arestvblog You should probably file this in the react-native-document-picker repo :) @qmatteoq FYI

**Comment by qmatteoq** [comment]
I'm sorry you're hitting this issue @arestvblog. I'll do a few investigations and get back to you.

**Comment by qmatteoq** [comment]
Hello @arestvblog, yesterday I pushed some changes which improve the Windows implementation and add a couple of new features to be better align with the Android / iOS counter part. For the moment they live in the [feat/use-bob-with-history](https://github.com/rnmods/react-native-document-picker/tree/feat/use-bob-with-history) branch, but they will be merged soon and trigger a new release on NPM.
I tried to reference directly that branch in a new RNW project (both in C# and C++) and it builds just fine, so hopefully it will fix the issue also for you once released!
